docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3"
+
+services:
+    db:
+        image: postgres
+        environment:
+            POSTGRES_DB: mariaquiteria
+            POSTGRES_USER: postgres
+            POSTGRES_PASSWORD: postgres
+        ports:
```

Comment:
Não precisa expor essa porta. Em produção isso seria uma porta aberta na internet pronta pra receber ataque de força de bruta ;) Se quiser essa porta aberta apenas local em dev, podemos fazer um docker-compose extends: https://docs.docker.com/compose/extends/

Reply:
O compose extends não é mais suportado nas versões mais novas. Vamos ter que mudar o version ali em cima.